docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -36,49 +33,44 @@ kubectl apply --filename serviceaccount.yaml
 ### Create Event Source for Kubernetes Events
 
 1. In order to receive events, you have to create a concrete Event Source for a
-   specific namespace. If you are wanting to consume events from a different
-   namespace or using a different `Service Account`, you need to modify the yaml
+   specific namespace. If you want to consume events from a different
+   namespace or use a different `Service Account`, you need to modify the yaml
```

Comment:
Can we specify what yaml file the user would need to modify to do that?

Reply:
Done.